storefront/pages/address/index.tsx

```diff
@@ -70,122 +72,182 @@ const Address: NextPage = () => {
     });
   }, []);
 
+  if (addresses.length == 0) {
+    return (
+      <>
+        <div className="container">
+          <div className="row">
+            <Head>
+              <title>Address</title>
```

Comment:
I believe there would be no need to render the entire page if there was no address. refer
![image](https://user-images.githubusercontent.com/62151844/231332469-6d3941d9-a818-45a7-831c-789956b9dc6b.png)

Reply:
It seems too much colorful